frontend/src/components/dialogs/SecretDialogGeneric.vue

```
@@ -101,26 +101,33 @@ export default {
       return !this.secret
     },
     secretData () {
-      return isObject(this.parsedYAML) ? this.parsedYAML : {}
+      return this.parsedYAML
     }
   },
   methods: {
     onInput (value) {
       this.$emit('input', value)
     },
-    async onChangeData () {
+    async parseYAML () {
       try {
         this.parsedYAML = await this.$yaml.load(this.data)
       } catch (err) {
-        this.parsedYAML = undefined
+        // Prevent errors on cosole for invalid yaml
```

Fix typo
```js
        // Prevent errors on console for invalid yaml
```
or change to our standard comment for ignoring error:
```js
     /* ignore errors */
```